### PR TITLE
Added new version to README

### PR DESCRIPTION
**Pull request by kimf**
Please give input here...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,83 +2,48 @@
...
+- Use an open https://developers.themoviedb.org/open source API
+- Discover most popular and highly rated movies
+- Display the movies with creative look and feel of an app to meet design guidelines for your platform (Material etc)
+- Launch a detail screen whenever a particular movie is selected
+- Allow user to save a favorite movie for offline access
+- Allow user to read movie reviews
 
 ## Assets
+
 You can find some icon assets at [https://github.com/erikflowers/weather-icons](https://github.com/erikflowers/weather-icons) that you can use if you please. You are more than welcome to use other assets that you feel are better fitted for your solution. We do not expect you to be a designer, but since application development on Daresay most often includes visual interfaces we do expect you to to be well aware of your platforms design guidelines and conventions.
```
Comment (likebobby) [inline README.md]: These icons are not of use for movies :)

File: README.md [hunk 2]
```diff
@@ -2,83 +2,48 @@
...
 
 ## Expectations
+
+We expect you to
+
+- Write clean code
+- Create a responsive design
+- Handle error cases
+
+Use of latest libraries and technologies such as Rx/coroutines etc and tested code is a big plus.
```
Comment (likebobby) [inline README.md:25]: remove the "such as" part?

**Review by likebobby — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kimf) [reply README.md]: Yeah removed that section :D

File: README.md [hunk 3]
```diff
@@ -2,83 +2,45 @@
...
-- Build an awesome weather app that shows the weather at your current  location.
+- Build an awesome movieapp that shows popular and high rated movies.
 - Code it for the platform (Android, iOS, web) you applied for or the one you prefer.
 
-## Minimum requirements
-- Show the weather for your current location
-- Use data from provided API (see API DOCS)
-- Good usability
-- Be visually appealing and follow platform conventions
-- Budget: 8-12 hours
```
Comment (likebobby) [inline README.md:13]: I like the budget part. But maybe we are communicating this in other ways?

**Review by kurtsson — COMMENTED** [review]
👍 Maybe suggesting creating a private fork? The previous work sample had so many public solutions that it was hard to know if they just copied someone elses. But now with unlimited private repos it might be feasible?

**Review by kurtsson — APPROVED** [review]

**Comment by mohamad-daresay** [comment]
> 👍 Maybe suggesting creating a private fork? The previous work sample had so many public solutions that it was hard to know if they just copied someone elses. But now with unlimited private repos it might be feasible?

How if we also ask for a commits log (avoiding squash), it can provide us a good vision of developer's thoughts and also preventing them from copying 🤔

**Review by mohamad-daresay — APPROVED** [review]
👍

**Comment by likebobby** [comment]
> > 👍 Maybe suggesting creating a private fork? The previous work sample had so many public solutions that it was hard to know if they just copied someone elses. But now with unlimited private repos it might be feasible?
> 
> How if we also ask for a commits log (avoiding squash), it can provide us a good vision of developer's thoughts and also preventing them from copying 🤔

Yes, great idea. We can add a section of "A nice commit history describing your work is preferred over squashing it into one."

**Review by likebobby — APPROVED** [review]
It looks good to me now.

**Pull request merged**